### DISCUSSION TITLE
External links in a table

### DISCUSSION THREAD

**Discussion by irt253** [Budibase/budibase#12240] [Help] [answered]
Is there any way in the existing basic tier to have users enter a url in a field form, and then save that information as a clickable link when they view the table? I am trying to do some basic linking with google drive and nothing I have found yet works :(

**Comment by ConorWebb96** [comment, marked as answer]
Hey @irt253,

You can add a button component to a table component and then use the `navigate to` action to link to whatever external urls you have stored within a table.

Please note this won't work in preview due to how this is done. This will only work for published apps. (I've supplied some screenshots below to help explain this better.)

<img width="476" alt="Screenshot 2023-11-01 at 07 14 50" src="https://github.com/Budibase/budibase/assets/126772285/0766c48d-3b25-4cc4-82ca-c18e3cedd850">
<img width="1437" alt="Screenshot 2023-11-01 at 07 15 08" src="https://github.com/Budibase/budibase/assets/126772285/4d818cf5-643f-457d-9680-a27d75ee5757">

I hope this helps.
